### PR TITLE
#9 HTTP 401 with preflight requests

### LINKED ISSUE #9: CORS issue when calling endpoints directly from my app
Hello,

I have CORS issue when trying to call the APIs from my local frontend app
image

image

while it is perfectly working with kcadm commands or directly with postman using the same user/brearer. I can't understand why it is not working directly inside my angular app (simple http get query with the bearer header)

I suspected something wrong with the "Web origins" config inside my client, because when I remove the host header in postman I got the same error
image

I tried multiple value in the  "Web origins" config without success.

I even tried with the realm "[multi-tenant](https://github.com/anarsultanov/keycloak-multi-tenancy/blob/main/src/test/resources/realm-export.json)" json on the test resources but got the same error.

Sorry, I may have missed something but can't figure out why I can't reach your endpoints. Have you any idea ?

- Keycloak version : 22.0.3
- keycloak-multi-tenancy version : 22.2.0

### PR DESCRIPTION
**Pull request by geoffreyfourmis**
resolve #9

### CODE REVIEW

File: src/main/java/dev/sultanov/keycloak/multitenancy/resource/CorsResource.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.services.resources.Cors;
+
+public class CorsResource {
+
+    private final KeycloakSession session;
+    private final HttpRequest request;
+
+    public CorsResource(KeycloakSession session, HttpRequest request) {
+        this.session = session;
```
Comment (anarsultanov) [inline src/main/java/dev/sultanov/keycloak/multitenancy/resource/CorsResource.java:16]: The session doesn’t seem to be used, why do we need it here?

File: src/main/java/dev/sultanov/keycloak/multitenancy/resource/CorsResource.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        this.request = request;
+    }
+
+    public static final String[] METHODS = {
```
Comment (anarsultanov) [inline src/main/java/dev/sultanov/keycloak/multitenancy/resource/CorsResource.java:20]: nit: Could constant be moved to the top before everything else?

**Review by anarsultanov — COMMENTED** [review]
Thank you for your contribution! Looks great, just a few minor comments.

**Comment by geoffreyfourmis** [comment]
Thx for your comments, I made the corrections

**Comment by anarsultanov** [comment]
Thanks for the prompt update @geoffreyfourmis!

**Pull request merged**